### PR TITLE
fixing a link that showed up as build error in docker/pinata/docs PR#…

### PR DESCRIPTION
**Pull request by londoncalling**
**\- What I did**

Fixed a link that is showing up as a docs build error in PR #4019  `docker/pinata/`

Signed-off-by: Victoria Bialas victoria.bialas@docker.com

### CODE REVIEW

**Comment by londoncalling** [comment]
@icecrime @sfsmithcha @SvenDowideit Can someone please merge this asap? It affects PR #4019 in `docker/pinata` needed for v1.12 docs by DockerCon.

**Comment by sfsmithcha** [comment]
LGTM. CI errors expected

**Comment by thaJeztah** [comment]
Actually, I think it was another link that showed as failing; see https://github.com/docker/docker/pull/23736, so I'm not sure we need this

File: docs/userguide/storagedriver/device-mapper-driver.md [hunk 1]
```diff
@@ -357,9 +357,7 @@ $ journalctl -fu dm-event.service
 If you run into repeated problems with thin pool, you can use the
 `dm.min_free_space` option to tune the Engine behavior. This value ensures that
 operations fail with a warning when the free space is at or near the minimum.
-For information, see <a
-href="../../../reference/commandline/dockerd/#storage-driver-options"
-target="_blank">the storage driver options in the Engine daemon reference</a>.
+For information, see [the storage driver options in the Engine daemon reference](../../reference/commandline/dockerd.md#storage-driver-options).
```
Comment (SvenDowideit) [inline docs/userguide/storagedriver/device-mapper-driver.md]: this isn't right - for `a href` links, we can't link to the `.md` file :( hugo doesn't process html links.

**Comment by SvenDowideit** [comment]
:+1:  for #23736

**Pull request closed without merging**